examples/lb/monitor/main.tf

```diff
@@ -0,0 +1,8 @@
+resource "vkcs_lb_monitor" "worker_ping_life_checker" {
+	name        = "worker-interface-life-checker-tf-example"
```

Comment:
make the name closes to resource name

Reply:
worker-ping-life-checker, or it can be shorter ping-lifeprobe